JDI.Light/JDI.Light/Elements/Complex/Table/Line.cs

```diff
@@ -18,23 +18,25 @@ public Line(List<string> list, List<string> headers)
             _headers = headers;
         }
 
-        public Line(List<UIElement> elements, List<string> headers)
+        public Line(IEnumerable<IWebElement> elements, List<string> headers)
         {
             _elements = elements;
             _headers = headers;
-            _data = ZipTwoLists(_headers, _elements.Select(el => el.Text).ToList());
+            _data = ZipTwoLists(_headers, _elements.Select(el => el.Text));
         }
 
-        private static Dictionary<string, string> ZipTwoLists(IReadOnlyCollection<string> keys, IReadOnlyCollection<string> values)
+        private static Dictionary<string, string> ZipTwoLists(IEnumerable<string> keys, IEnumerable<string> values)
```

Comment:
please do not create a static members inside a class that can have several instances